app/src/main/java/org/openobservatory/ooniprobe/activity/MainActivity.java

```diff
@@ -129,6 +136,45 @@ public void onDrawerStateChanged(int newState){
         }
 
         checkInformedConsent();
+
+        // XXX: This is probably not correct: we would like to send
+        // info to the orchestrator only when the network or any other
+        // orchestrator parameter like country code changed.
```

Comment:
@lorenzoPrimi do we want to keep this as part of the release? I added it as a ghetto solution for always refreshing the code, but maybe it's not the case? If we want to keep it, I can zap the `XXX` comment.